lib/src/utils/duration.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+import 'package:lichess_mobile/src/utils/l10n_context.dart';
+
+extension DurationExtensions on Duration {
+  String toDaysHoursMinutes(BuildContext context) {
+    /*
```

Comment:
same remark about useless comment in the code that could be a GitHub comment.

Reply:
I wasn't aware of GitHub comments, sorry!